js/build.js

```diff
@@ -356,6 +365,7 @@ Fliplet.Widget.instance('chat', function (data) {
 
   function closeConversation(clickedChat) {
     $wrapper.removeClass('chat-open');
+    $wrapper.css('top', '');
```

Comment:
Take this line and wrap it in a function.

This way it's clear when and how the UI enters and exits fullscreen.

```suggestion
    exitChatFullScreen();
```

Reply:
@tonytlwu I've tried to use CSS solution.
Chat overlay already has position absolute like all components in it. And if try to change it to the fixed we still need to change the top position.